src/view/FilesAppIntegration.js

```diff
@@ -261,7 +261,8 @@ export default {
 
 	_addHeaderShareButton() {
 		if ($('header').length) {
-			const $button = $('<div id="richdocuments-sharing"><a class="icon-shared icon-white"></a></div>')
+			const isInverted = window.OCA.Theming.inverted
```

Comment:
To be on the safe side if the theming app is disabled:

```suggestion
			const isInverted = Boolean(window?.OCA?.Theming?.inverted)
```